charts/kof-storage/templates/victoria/vmalert.yaml

```
@@ -24,6 +24,8 @@ spec:
     url: {{ .Values.victoriametrics.vmalert.remoteRead | default (printf "http://vmselect-cluster.%s.svc:8481/select/0/prometheus" .Release.Namespace) }} 
   remoteWrite:
     url: http://vminsert-cluster.{{ .Release.Namespace }}.svc:8480/insert/0/prometheus/api/v1/write
+  replicaCount: {{ .Values.victoriametrics.vmalert.replicaCount | default 2 }}
```

```suggestion
  replicaCount: {{ .Values.victoriametrics.vmalert.replicaCount }}
```
We don't encourage to have default values in templates, please create the default value in values.yaml and reference it without default function

---

done